py/gc.c

```diff
@@ -693,34 +693,6 @@ void gc_dump_alloc_table(void) {
         int c = ' ';
         switch (ATB_GET_KIND(bl)) {
             case AT_FREE: c = '.'; break;
-            /* this prints out if the object is reachable from BSS or STACK (for unix only)
```

Comment:
No, I like your code simplification :)  This code here will change to scan the root pointers using the global state structure, not using the bss symbols.  Therefore it'll become available on mac/windows as well, even with your code removal.